src/WebJobs.Script.WebHost/Models/HostAssignmentContext.cs

```diff
@@ -28,6 +29,9 @@ public class HostAssignmentContext
         [JsonProperty("CorsSpecializationPayload")]
         public CorsSettings CorsSettings { get; set; }
 
+        [JsonProperty("Secrets")]
+        public FunctionAppSecrets Secrets { get; set; }
```

Comment:
Linux Consumption specialization doesn't currently send this along, but I've plumbed this all the way through so that if it does, it'll just work.